backend/infrahub/actions/triggers.py

```diff
@@ -0,0 +1,21 @@
+from infrahub.events.node_action import NodeCreatedEvent, NodeDeletedEvent, NodeUpdatedEvent
+from infrahub.trigger.models import BuiltinTriggerDefinition, EventTrigger, ExecuteWorkflow
+from infrahub.workflows.catalogue import CONFIGURE_ACTION_RULES
+
+from .constants import NODES_THAT_TRIGGER_ACTION_RULES_SETUP
+
+TRIGGER_ACTION_RULE_UPDATE = BuiltinTriggerDefinition(
```

Comment:
We have a similar setup for the Webhooks would be good to keep them aligned as much as possible